mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -127,6 +135,10 @@ CLASS none/sg net/minecraft/entity/Entity
 	METHOD a damage (Lnone/rr;F)Z
 		ARG 0 source
 		ARG 1 damage
+	METHOD a (Lnone/sg;)V
+	METHOD a startRiding (Lnone/sg;Z)Z
```

Comment:
We're going to start riding an entity then set the entity ridden. startRiding makes more sense.